node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/component/Environment.java

```diff
@@ -290,6 +277,7 @@ public Path pathInNodeUnderVespaHome(String relativePath) {
         private Optional<String> coredumpFeedEndpoint = Optional.empty();
         private NodeType nodeType = NodeType.tenant;
         private String defaultFlavor;
+        private String cloud = "";
```

Comment:
This variable must be set, so leave it at `null` and `requireNonNull()` in `build()`.

All the `requireNonNulls()` should probably be in the constructor instead though...

Reply:
The idea may have been to leave the cloud at "" by default, and only set it in hosted for now?

Reply:
Thanks, fixed as suggested